backend/services/interfaces/classService.ts

```diff
@@ -0,0 +1,99 @@
+import { UserDTO, Grade } from "../../types";
+import { TestSessionResponseDTO } from "./testSessionService";
+
+export interface StudentRequestDTO {
+  firstName: string;
+  lastName: string;
+  studentNumber?: string;
+}
+
+export interface StudentResponseDTO {
+  id: string;
+  firstName: string;
+  lastName: string;
+  studentNumber?: string;
+}
+
+export interface ClassRequestDTO {
+  className: string;
+  schoolYear: number;
+  gradeLevel: Grade[];
+  teacher: string;
+  testSessions: string[];
+  students?: StudentRequestDTO[];
+}
+
+export interface ClassResponseDTO {
+  id: string;
+  className: string;
+  schoolYear: number;
+  gradeLevel: Grade[];
+  teacher: UserDTO;
+  testSessions: TestSessionResponseDTO[];
+  students: StudentResponseDTO[];
+}
+
+export interface IClassService {
+  /**
+   * This method creates a new class in the database.
+   * @param classObj  new class
+   * @returns the created class
+   * @throws Error if creation fails
+   */
+  createClass(classObj: ClassRequestDTO): Promise<ClassResponseDTO>;
```

Comment:
```suggestion
  createClass(class: ClassRequestDTO): Promise<ClassResponseDTO>;
```
can we rename all occurrences of `classObj` to `class` for consistency with other naming?

Reply:
hello, so I set it to classObj since class is a forbidden parameter name ;( is there any other suggestions for a name?

Reply:
ah in that case just leave as is!